src/core/Class.js

```diff
@@ -60,6 +60,13 @@ export class Class {
 		return this;
 	}
 
+	// @function setDefaultOptions(options: Object): this
+	// Configures the [default `options`](#class-options) on the prototype of this class.
+	static setDefaultOptions(options) {
+		Util.setOptions(this.prototype, options);
+		return this;
+	}
+

```

Comment:
I could not find a more concise way to set the default options including prototype inheritance with the new ES6 class syntax. A new function is required, since neither `include` not `mergeOptions` use `Object.create` to enable prototype inheritance (and tests explicitly test for this).

I also tried/considered `Map.prototype.options= {...}` after `class Map extends Evented {...}`, but then the options hidden at the bottom of the source files.

Reply:
Does it not work to define static options in the classes kinda like before, and then update the setOptions util to look at them? Some variation of the below? Note the obj.constructor?.options is what gets the static options defined on the real classes?

```js
export function setOptions(obj, options) {
	const baseOptions = obj.options || obj.constructor?.options || null;

	if (!Object.hasOwn(obj, 'options')) {
		obj.options = baseOptions ? Object.create(baseOptions) : {};
	}
	for (const i in options) {
		if (Object.hasOwn(options, i)) {
			obj.options[i] = options[i];
		}
	}
	return obj.options;
}
```